install.sh

```diff
@@ -193,7 +193,7 @@ echo_out 'Set up the local package repo...'
 # Download the chromebrew repository.
 curl_wrapper -L --progress-bar https://github.com/"${OWNER}"/"${REPO}"/tarball/"${BRANCH}" | tar -xz --strip-components=1 -C "${CREW_LIB_PATH}"
 
-BOOTSTRAP_PACKAGES='lz4 zlib xzutils zstd crew_mvdir ruby git ca_certificates libyaml openssl'
+BOOTSTRAP_PACKAGES='lz4 zlib xzutils zstd zlib_ng crew_mvdir ruby git ca_certificates libyaml openssl'
```

Comment:
Do we need both zlib and zlib_ng?

Reply:
Zlib_ng has a compat mode, but I don't want to rebuild every single program that would require and having a separate library without a conflicting file name also keeps us from having conflicts with the base system libraries.

Reply:
Yes, they create separate libraries unless the zlib_ng package is rebuilt with compatibility mode, which may be its own headache we should avoid, at least for now.

Reply:
(Replacing zlib with zlib-ng may require rebuilds of packages, and if you remember the jpeg/jpeg_turbo fiasco, let's not jump into that so quickly.)

Reply:
Does git fail when zlib_ng is removed? If its only slower using zlib as a fallback, then it doesn't need to be an essential package.

Reply:
Also, it would be ideal if we could get bootstrap packages to NOT depend upon the system zlib, so that system updates (or older versions on EOL systems) don't break our binaries.

Reply:
> Does git fail when zlib_ng is removed? If its only slower using zlib as a fallback, then it doesn't need to be an essential package.

That's a really good question. My rationale is to conservatively set it as an essential package to begin with, to ensure we don't break anything, and then back it out of essential packages if it doesn't break git. The last thing we want is a broken git since our updates depend upon it.